grades.js

```diff
@@ -0,0 +1,51 @@
+var scores = [82, 71, 95, 55, 98, 69, 72, 78, 84, 64, 58, 87];
```

Comment:
I know I told you before but really great job with indenting. I was not doing that this early in the game!